x-pack/platform/plugins/shared/fleet/public/applications/integrations/sections/epm/screens/home/hooks/use_available_packages.tsx

```diff
@@ -105,11 +105,22 @@ const packageListToIntegrationsList = (packages: PackageList): PackageList => {
         })
       : [];
 
-    const tiles = filterPolicyTemplatesTiles<PackageListItem>(
+    let tiles = filterPolicyTemplatesTiles<PackageListItem>(
       pkg.policy_templates_behavior,
       topPackage,
       integrationsPolicyTemplates
     );
+
+    // remove tiles that only have data streams of excluded types
+    tiles = tiles.filter((tile) => {
+      return (
+        !tile.integration ||
+        tile.data_streams.some(
+          (dataStream) => dataStream.dataset === `${tile.name}.${tile.integration}`
```

Comment:
Do we need to filter on the data stream's `type` or am I misremembering how these titles are structured?